eng/Signing.props

```diff
@@ -16,7 +16,7 @@
     <FileSignInfo Include="Verify.dll" CertificateName="3PartySHA2" />
     <FileSignInfo Include="Verify.Xunit.dll" CertificateName="3PartySHA2" />
     <FileSignInfo Include="EmptyFiles.dll" CertificateName="3PartySHA2" />
-    <FileSignInfo Include="FluentAssertions.dll" CertificateName="3PartySHA2" />
```

Comment:
Are we actually signing these files and redisting them? If so, we'll need to update the TPN files to remove FluentAssertions and replace it with AwesomeAssertions.

Reply:
Reviewed TPN files, didn't see anything for FA, so we don't need to add AA either